src/common/telemetry/libjson/TelemetryJson.hpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef TELEMETRYJSON_HPP
+#define TELEMETRYJSON_HPP
+
+#define TELEMETRY_BINARY_NAME "OSConfigTelemetry"
+
+#ifdef __cplusplus
+
+#include <string>
+#include <memory>
+
+// C++ interface
+namespace TelemetryJson
+{
+
+class Logger
+{
+public:
+    Logger();
+    ~Logger();
+
+    // Disable copy constructor and assignment operator
+    Logger(const Logger&) = delete;
+    Logger& operator=(const Logger&) = delete;
+
+    // Enable move constructor and assignment operator
+    Logger(Logger&& other) noexcept;
+    Logger& operator=(Logger&& other) noexcept;
+
+    // @brief Opens the logger for writing
+    // @return true on success, false on failure
+    bool open();
+
+    // @brief Closes the logger
+    // @return true on success, false on failure
+    bool close();
+
+    // @brief Logs an event with key-value pairs
+    // @param eventName Name of the event to log
+    // @param keyValuePairs Array of key-value pair strings (must be even number of elements)
+    // @param pairCount Number of key-value pairs (keyValuePairs array size / 2)
+    // @return true on success, false on failure
```

Comment:
These comments here are also non compliant with our coding style. Please see the other comment on this. Can you please fix this? Thanks